### ISSUE TITLE
combobox "selected" not announced on item with aria-selected

### ISSUE THREAD

**Issue by aFrontendDev** [nvaccess/nvda#17195]
### Steps to reproduce:
Open link to w3c combo box - https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only/
tab to the example combo box with the label "Favorite Fruit"
Open box & select an option
Close box, return to it and open
It will highlight the selected option visually

### Actual behavior:

Opening the combo box that has a selected option will result in that option being focused and visually highlighted
NVDA will read out the selected option, great
It will not specify that it is "selected"

### Expected behavior:

Opening the combo box that has a selected option will result in that option being focused and visually highlighted
NVDA will read out the selected option, great
It should  specify that it is "selected", e.g. "Banana selected 3 of 13" as other screen readers do such as osx voice over

#### NVDA installed/portable/running from source:
NVDA installed

#### NVDA version:
2024.3.1 (2024.3.1.33894)

#### Windows version:
Windows 11 Pro 22H2

#### Name and version of other software in use when reproducing the issue:
Chrome Version 129.0.6668.59

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by Adriani90** [comment]
the actual behavior is expected in this case. Hearing „selected“ is too much verbosity. Note that e.g. in Windows Explorer when arrowing around the selected file or folder is also announced without „selected“.
It is rather important to hear when something is not selected while it is focused. But this happens already with NVDA.
Closing as won‘t fix.
If you have any other arguments, please comment and we can reopen. But see also the discussion in #11986 and #16449.
